scripts/template/src/patternfly/{folder}/{name}/examples/{name}.md

```diff
@@ -1,8 +1,7 @@
 ---
-title: '{{nameCapitalized}}'
-section: beta
```

Comment:
Do we not need `section` in the meta data anymore?

Reply:
We do! Added back as `{folder}`.